examples/job/ml-train-model-predict-properties.py

```diff
@@ -148,8 +160,12 @@
 # In[]:
 
 
-owner_id = material_endpoints.list({"isDefault": True, "owner._id": ACCOUNT_ID})[0]["owner"]["_id"]
-project_id = project_endpoints.list({"isDefault": True, "owner._id": ACCOUNT_ID})[0]["_id"]
+owner_id = material_endpoints.list({"isDefault": True, "owner._id": ACCOUNT_ID})[0][
+    "owner"
+]["_id"]
+project_id = project_endpoints.list({"isDefault": True, "owner._id": ACCOUNT_ID})[0][
+    "_id"
```

Comment:
Same as above

Reply:
It seems that this also has to do with the autoformatter's default line length of 88. I could extend it to 100, or something like that.